contracts/Schema.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.6.0 <0.9.0;
```

Comment:
Probably version range should be more strict. What is the version of solidity compiler we are using?

Reply:
current truffle setup compiles all contracts with 0.6.0

Reply:
changed to 0.6.0